CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.9.0] - 2016-12-21
+~~~~~~~~~~~~~~~~~~~~
+
+Added
+-----
+
+* Added alphabetical ordering for catalog drop down and displayed catalog details link next to selected catalog.
```

Comment:
Please mention `django admin` so that we are clear where we are making these changes.